schema.go

```diff
@@ -82,6 +82,56 @@ func (sch *Schema) Apply(migrations []Migration) (n int, err error) {
 	return n, nil
 }
 
+// ApplyEach applies each migrations in a separate transaction. It returns the number
+// of applied migrations and error if any.
+func (sch *Schema) ApplyEach(migrations []Migration) (n int, err error) {
+	now := time.Now()
+	q := `INSERT INTO "` + sch.schemaName + `"` + `."` + sch.migTableName + `" (name, applied_at) ` +
+		`VALUES ($1, $2)`
+	for _, m := range migrations {
+		err := func() (err error) {
+			var tx *sql.Tx
```

Comment:
`tx` could be declared once at the top level.

Reply:
@ainar-g Done